src/components/LessonState/index.js

```diff
@@ -10,7 +10,7 @@ export default ({lesson}) => {
         mb3 ttu tc pv2 ph3 br2 ba b--dashed dib
         ${statusColor} b--${statusColor}
       `}>
-        {lesson.state}
+        {lesson.state === 'rejected' ? 'Changes Requested' : lesson.state}
```

Comment:
It would be good to keep this in `src/utils/lessonStates` with all the other lesson state related information; could introduce a new `title` key on the `detailsByLessonState` objects. But nbd.